core/conversion/converters/converters.h.orig

```diff
@@ -0,0 +1,46 @@
+#pragma once
```

Comment:
Do we still need this file?

Reply:
odd, not sure how this file got into the commit - I made sure to leave it out. we can delete it, I'll do that